Phenotyping thumbnail and thumbnail documentation (#509)
alan-turing-institute/scivision: docs/data_repository_template.md

You should include a `LICENSE` file in the repository, so that scivision users who come across it can understand the conditions of the data's usage. For help deciding which license to include, check out the section on [data licenses](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-data.html) in The Turing Way online handbook.

## 🎆 Data Thumbnail

When viewing the dataset in [sci.vision](https://sci.vision/#/datasource-grid) a dataset thumbnail is required. The thumbnail can be an interesting image or slice of the dataset that catches the attention of the Scivision user (creativity encouraged!). 

- It must be a 256x256 PNG or JPEG file
- The file name should be the name of the dataset as in the catalog

> Once you have identified a suitable thumbnail please upload it to the [thumbnail datasources folder](https://github.com/alan-turing-institute/scivision/tree/main/scivision/catalog/data/thumbnails/datasources). 

Reviewer: ```suggestion
Once you have identified a suitable thumbnail, follow the :ref:`how-to-contribute` guide to open a pull request to the Scivision GitHub repo which adds the thumbnail image in the `scivision/catalog/data/thumbnails/datasources` folder. 
```

You should include a `LICENSE` file in the repository, so that scivision users who come across it can understand the conditions of the data's usage. For help deciding which license to include, check out the section on [data licenses](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-data.html) in The Turing Way online handbook.

> <<<<<<< HEAD

Reviewer: ```suggestion
```
Removing these git conflict markers

.. _requirements-for-scivision:

## 💻 Requirements for scivision website

.. _data-thumbnail:

> ## 🎆 Data Thumbnail

Reviewer: ```suggestion
### 🎆 Data Thumbnail
```
Make this header one size smaller

When viewing the dataset in [sci.vision](https://sci.vision/#/datasource-grid) a dataset thumbnail is required. The thumbnail can be an interesting image or slice of the dataset that catches the attention of the Scivision user (creativity encouraged!). 

- It must be a 256x256 PNG or JPEG file
- The file name should be the name of the dataset as in the catalog

> <<<<<<< HEAD

Reviewer: ```suggestion
```

> Once you have identified a suitable thumbnail please upload it to the [thumbnail datasources folder](https://github.com/alan-turing-institute/scivision/tree/main/scivision/catalog/data/thumbnails/datasources). 

Reviewer: ```suggestion
```
Removed in favour of the version below. @Tonks684 basically the conflicted merge you had before has both versions separated by "conflict markers". So what you can do when this happens in future is remove all of the "conflict markers" and just keep the text (or code) that you want from the two branches you are merging. For now I am removing these on the PR here - I suggest you:

1. accept all my changes
2. run `git pull` to get the changes locally (I assume you haven't made any subsequent commits since this last push)
3. check the documentation looks right and if not, make some new changes and push them again

> =======

Reviewer: ```suggestion
```
This is another conflict marker, it separates the two conflicting parts of the code

Once you have identified a suitable thumbnail, follow the :ref:`how-to-contribute` guide to open a pull request to the Scivision GitHub repo which adds the thumbnail image in the `scivision/catalog/data/thumbnails/datasources` folder. 

> >>>>>>> e9be9e1be72d6db9ec0f9b2505f36e08533d5a8c

Reviewer: ```suggestion
```
